src/seedu/addressbook/data/person/Address.java

```diff
@@ -12,7 +12,10 @@
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
-    public final String value;
+    public final Block block;
```

Comment:
Try to make it private. In general, start off with private members, then use protected or public only if absolutely necessary.